eox_tenant/edxapp_wrapper/auth.py

```diff
@@ -0,0 +1,15 @@
+"""
+Auth definitions.
+"""
+
+from importlib import import_module
+from django.conf import settings
+
+
+def get_edx_auth_failed():
+    """ Gets GeneratedCertificate model. """
```

Comment:
As the [standard](https://www.python.org/dev/peps/pep-0257/#id16) describes: 

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

So the (s) or conjugated verbs are not necessary.

Reply:
ohohohohohoh